modules/canbus/vehicle/gem/protocol/date_time_rpt_83.cc

```diff
@@ -31,34 +31,48 @@ Datetimerpt83::Datetimerpt83() {}
 const int32_t Datetimerpt83::ID = 0x83;
 
 void Datetimerpt83::Parse(const std::uint8_t* bytes, int32_t length,
-                         ChassisDetail* chassis) const {
-  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_time_second(time_second(bytes, length));
-  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_time_minute(time_minute(bytes, length));
-  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_time_hour(time_hour(bytes, length));
-  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_date_day(date_day(bytes, length));
-  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_date_month(date_month(bytes, length));
-  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_date_year(date_year(bytes, length));
+                          ChassisDetail* chassis) const {
+  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_time_second(
+      time_second(bytes, length));
+  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_time_minute(
+      time_minute(bytes, length));
+  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_time_hour(
+      time_hour(bytes, length));
+  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_date_day(
+      date_day(bytes, length));
+  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_date_month(
+      date_month(bytes, length));
+  chassis->mutable_gem()->mutable_date_time_rpt_83()->set_date_year(
+      date_year(bytes, length));
 }
 
-// config detail: {'name': 'time_second', 'offset': 0.0, 'precision': 1.0, 'len': 8, 'is_signed_var': False, 'physical_range': '[0|60]', 'bit': 47, 'type': 'int', 'order': 'motorola', 'physical_unit': 'sec'}
-int Datetimerpt83::time_second(const std::uint8_t* bytes, int32_t length) const {
+// config detail: {'name': 'time_second', 'offset': 0.0, 'precision': 1.0,
+// 'len': 8, 'is_signed_var': False, 'physical_range': '[0|60]', 'bit': 47,
+// 'type': 'int', 'order': 'motorola', 'physical_unit': 'sec'}
+int Datetimerpt83::time_second(const std::uint8_t* bytes,
+                               int32_t length) const {
   Byte t0(bytes + 5);
   int32_t x = t0.get_byte(0, 8);
 
   int ret = x;
   return ret;
 }
 
-// config detail: {'name': 'time_minute', 'offset': 0.0, 'precision': 1.0, 'len': 8, 'is_signed_var': False, 'physical_range': '[0|60]', 'bit': 39, 'type': 'int', 'order': 'motorola', 'physical_unit': 'min'}
-int Datetimerpt83::time_minute(const std::uint8_t* bytes, int32_t length) const {
+// config detail: {'name': 'time_minute', 'offset': 0.0, 'precision': 1.0,
+// 'len': 8, 'is_signed_var': False, 'physical_range': '[0|60]', 'bit': 39,
+// 'type': 'int', 'order': 'motorola', 'physical_unit': 'min'}
+int Datetimerpt83::time_minute(const std::uint8_t* bytes,
+                               int32_t length) const {
   Byte t0(bytes + 4);
   int32_t x = t0.get_byte(0, 8);
 
```

Comment:
can we change template to return t0.get_byte(0, 8)?

Reply:
@startcode Will do